doc/lua_api.txt

```diff
@@ -3758,7 +3758,9 @@ It can be created via `Settings(filename)`.
 
 #### Methods
 * `get(key)`: returns a value
-* `get_bool(key)`: returns a boolean
+* `get_bool(key, default)`: returns a boolean
```

Comment:
`[default]`, as done in other functions with optional parameters.

Reply:
Thanks, I'll fix that.